crates/net/downloaders/src/file_client.rs

```diff
@@ -264,6 +264,12 @@ impl<B: FullBlock<Header: reth_primitives_traits::BlockHeader>> FromReader
                     parent_header = Some(sealed);
                 }
 
+                // Validate block against header
+                self.consensus.validate_block_pre_execution(&SealedBlock::new_unchecked(
+                    block.clone(),
+                    block_hash,
+                ))?;
```

Comment:
nit: can we remove the clone here by creating the `SealedBlock` first, then referring to it later instead of doing `block.header().number()` for example?